### PR TITLE
Sysctl saving

### PR DESCRIPTION
**Pull request by rapenne-s**
This commit permit the following :

sysctl "net.ipv4.ip_forward" => 1, persistent => "true";

so net.ipv4.ip_forward will be set to 1 and net.ipv4.ip_forward=1 will be added/updated in /etc/sysctl.conf

### CODE REVIEW

File: lib/Rex/Commands/Sysctl.pm [hunk 1]
```diff
@@ -79,6 +94,11 @@ sub sysctl {
       Rex::Logger::debug("$key has already value $val");
     }
 
+    if ( exists $options{persistent} && $options{persistent} eq "true" ) {
```
Comment (krimdomu) [inline lib/Rex/Commands/Sysctl.pm]: @rapenne-s maybe we can just do `if ( exists $options{persistent} && $options{persistent} ) {` there, so people can also use `persistent => 1` to activate this.

Comment (rapenne-s) [inline lib/Rex/Commands/Sysctl.pm]: I was thinking about this but I think I should add a way to remove the sysctl from /etc/sysctl.conf.

That could be changed to 

```
if ( exists $options{persistent} )
  if ( $options{persistent} )
    sysctl_save;
  else
    sysctl_remove;
}
```
Better ?

Comment (ferki) [inline lib/Rex/Commands/Sysctl.pm]: I think `sysctl_remove` should only be called if `persistent` is FALSE.

As a somewhat related topic, I would find `ensure => 'present'` and `ensure => 'absent'` more clear about what it means and does, plus more in-line with the usual rex syntax for other things.

**Comment by ferki** [comment]
@rapenne-s: please rebase this on top of current development branch, and push again (even forced, if needed). Unfortunately the development branch had a test failure at the time when you forked your feature branch off, and that's why tests fail now here.

ps.: what do you think about having `ensure => 'present'` and `ensure => 'absent'` instead of `persistent`?

**Comment by krimdomu** [comment]
Hi @rapenne-s 

thank you also for this patch :+1: 

I've just merged it into development and development-1.x branch.
I modified it a bit.

I changed 
```perl
if ( exists $options{persistent} && $options{persistent} eq "true" ) {
```
to
```perl
if ( $options{persistent} ) {
```

**Pull request closed without merging**